spec/components/topic_view_spec.rb

```diff
@@ -694,4 +694,23 @@ def topic_view_near(post, show_deleted = false)
       expect(topic_view.last_post_id).to eq(p3.id)
     end
   end
+
+  describe '#read_time' do
+    let!(:post) { Fabricate(:post, topic: topic) }
+
+    before do
+      post.revise(Discourse.system_user, raw: "![image|100x100](upload://upload.png)")
+      topic_view.topic.reload
+    end
+
+    it 'should return the right read time' do
+      expect(topic_view.read_time).to eq(1)
```

Comment:
This test is testing default values and is bound to fail whenever we change them.